geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -249,22 +250,22 @@ private void startHttpService(boolean isServer) {
 
           if (agentUtil.isWebApplicationAvailable(gemfireWar)) {
             this.httpServer = JettyHelper.addWebApplication(this.httpServer, "/gemfire", gemfireWar,
-                securityService);
+                securityService, null);
             this.httpServer = JettyHelper.addWebApplication(this.httpServer, "/geode-mgmt",
-                gemfireWar, securityService);
+                gemfireWar, securityService, null);
           }
 
           if (agentUtil.isWebApplicationAvailable(pulseWar)) {
-            this.httpServer =
-                JettyHelper.addWebApplication(this.httpServer, "/pulse", pulseWar, securityService);
+            this.httpServer = JettyHelper.addWebApplication(this.httpServer, "/pulse", pulseWar,
+                securityService, createSslProps());
```

Comment:
you don't need to create the sslProps again, on line 248, an SSLConfig object is already available by calling SSLConfigurationFactory.getSSLConfigForComponent(config, SecurableCommunicationChannel.WEB), you can just use that to pass to the addWebApplication call.